pootle/apps/pootle_store/models.py

```diff
@@ -1972,10 +1969,12 @@ def update(self, overwrite=False, store=None, only_newer=False, user=None,
                                                        update_unitids, user,
                                                        revision=update_revision,
                                                        submission_type=submission_type)
-            self.file_mtime = disk_mtime
+            self.file_mtime = self.get_file_mtime() or self.file_mtime
 
-            if (filter(lambda x: changes[x] > 0, changes) and
-                store == self.file.store):
+            update_sync_revision = (filter(lambda x: changes[x] > 0, changes)
+                                    and self.file
```

Comment:
I'm not arguing what's more readable; just asking for the change [to be consistent](http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#if-statements).